builder/proxmox/step_start_vm.go

```diff
@@ -89,7 +89,7 @@ func (s *stepStartVM) Run(ctx context.Context, state multistep.StateBag) multist
 	state.Put("vmRef", vmRef)
 	// instance_id is the generic term used so that users can have access to the
 	// instance id inside of the provisioners, used in step_provision.
-	state.Put("instance_id", vmRef)
+	state.Put("instance_id", vmRef.VmId())
```

Comment:
For the future us who wonder about this, let's add a small mental note :) 
```suggestion
	// Note that this is just the VMID, we do not keep the node, pool and other
	// info available in the vmref type.
	state.Put("instance_id", vmRef.VmId())
```

Reply:
ah that was it, really cool fix !